src/modules/yutobot/index.js

```diff
@@ -5,6 +5,14 @@ const owoifier = require("../owoifier");
 const pkgInfo = require("../../../package.json");
 const simpleCmdDict = require("./simple_call_response.json");
 
+const { registerFont, createCanvas, loadImage } = require('canvas')
+registerFont('./assets/FOT-RodinBokutohPro-DB.otf', { family: 'RodinBokutohPro' })
+const width = 1920;
+const height = 1080;
+const canvas = createCanvas(width, height)
+const ctx = canvas.getContext('2d')
+ctx.font = '75px "FOT-RodinBokutoh Pro DB"';
+
```

Comment:
Can you move this into a different script under the `modules` folder so that the main script here only has an import line (see the other `require()` statements above)?